### ISSUE TITLE
Use column remarks/comments as initial descriptions when syncing new database

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#3089]
Some databases let you add remarks to columns. Not sure how many of our users this would affect but it's another piece of potentially useful information we could pull in. 

[Unfortunately it doesn't look like there's a standard way to do this in JDBC](http://stackoverflow.com/questions/37612183/how-to-get-column-comments-in-jdbc/37612403) so it will probably have to be done on a driver-by-driver basis. 

⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by camsaul** [comment]
For example here's what PgAdmin does with remarks

```sql
CREATE TABLE schema_permissions
(
  id serial NOT NULL,
  group_id integer NOT NULL,
  database_id integer NOT NULL,
  schema character varying(256) NOT NULL,
  unrestricted_table_access boolean NOT NULL, -- Is this group allowed to access any table belonging to this schema? (As opposed to only having access to a whitelisted subset of tables.)
  CONSTRAINT pk_schema_permissions PRIMARY KEY (id),
  CONSTRAINT fk_schema_permissions_database_id FOREIGN KEY (database_id)
      REFERENCES metabase_database (id) MATCH SIMPLE
      ON UPDATE NO ACTION ON DELETE NO ACTION,
  CONSTRAINT fk_schema_permissions_group_id FOREIGN KEY (group_id)
      REFERENCES permissions_group (id) MATCH SIMPLE
      ON UPDATE NO ACTION ON DELETE NO ACTION,
  CONSTRAINT schema_permissions_group_id_database_id_schema_key UNIQUE (group_id, database_id, schema)
);
```

**Comment by camsaul** [comment]
Note to self:

For Oracle looks like we need to enable this by setting the `remarksReporting` property. See https://docs.oracle.com/cd/E11882_01/appdev.112/e13995/oracle/jdbc/OracleDriver.html

**Comment by asovak** [comment]
Yes, it would be extremely effective to allow database to pull description for tables from database remarks/comments. Analytics/developers who create a database are the best people to describe the data. In best architectures, this description is stored and manages within tables in a database. The Metabase "description" layer should reflect that. Please, develop this feature.

**Comment by EvilPuppetMaster** [comment]
I have a working implementation of this for H2. Because of the inconsistent JDBC support, it defaults to doing nothing if drivers don't explicitly implement. I intend to test at least on Postgres, MySql. And then I'll submit for review.

**Comment by camsaul** [comment]
Cool @EvilPuppetMaster, thanks!

**Comment by EvilPuppetMaster** [comment]
I've had a few attempts at getting this ready to contribute, but I'm having lots of problems with the unit tests. There appear to be loads of errors even when I run the tests on an unchanged vanilla master branch. However the test assertions all still pass.

Looking at the history of builds in CircleCI, I think all the errors I am seeing are happening in the CI builds as well. But it's hard to tell. 

Is this just what the current state of the unit tests is? Or is it possible I have something wrong with my environment?

**Comment by EvilPuppetMaster** [comment]
@camsaul PR submitted (see above). I can't request specific reviewers, do you guys just go through them as time permits?

**Comment by camsaul** [comment]
Implemented by #6974
